setup.py

```diff
@@ -33,7 +33,7 @@
             "defusedxml",
             "svg.path",
             "Pillow>=6.2.2",  # minimum version tested there: https://github.com/PyFPDF/fpdf2/actions/runs/2295868575
-            "fonttools",
+            "fonttools==4.37.1",  # later versions may work but not tested.
```

Comment:
```suggestion
            "fonttools>=4.37.1",
```
An article that explains why versions ranges are better for libraries: https://medium.com/meilleursagents-engineering/building-and-managing-python-libraries-ebf4f805be6e#08a5

Don't get me wrong: for end-user applications, dependency pinning is **great**.
But for a library, we want to offer our users some flexibility 😊

Reply:
Fair enough, I was kind of expecting you would not accept this.
I had not chacked the minimal version of fontTools but after some checks, the FontTools version need be at least 4.30.0 for the error seen in #524 to not be triggered.

Thanks for the article, now I'm wondering though, why not capping the version?

Reply:
> I'm wondering though, why not capping the version?

For the same reason we (and most libraries) do not cap their dependencies to a maximum version:
be cause we trust that they will be [backward compatible](https://en.wikipedia.org/wiki/Backward_compatibility) in the future.